packages/apps/reputation-oracle/server/src/modules/user/user.service.ts

```diff
@@ -256,14 +255,18 @@ export class UserService {
     }
 
     const signer = this.web3Service.getSigner(chainId);
-
     const kvstore = await KVStoreClient.build(signer);
 
-    const status = await KVStoreUtils.get(
-      chainId,
-      signer.address,
-      user.evmAddress,
-    );
+    let status: string | undefined;
+    try {
+      status = await KVStoreUtils.get(chainId, signer.address, user.evmAddress);
+    } catch (error) {
+      this.logger.error(
+        `Error fetching operator status for address ${user.evmAddress}: ${error.message}`,
```

Comment:
The error message already contains the address
```suggestion
        `Error fetching operator status: ${error.message}`,
```
You can use a constant for "`Error fetching operator status"